styles.css

```diff
@@ -95,3 +95,268 @@ div.box {
+  box-sizing: border-box;
+  padding-right: 16px;
+  padding-left: 16px;
+  padding-bottom: 10px;
+  color: #fff;
+  background: #1a2236;
+  height: 50px;
+  justify-content: center;
+  text-decoration: none;
+}
+
+.second-text {
+  font-family: 'Poppins', sans-serif;
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 21px;
+  color: #1a2236;
+  height: 135px;
+  margin: 20px;
+}
+
+.lfs-container {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-rows: auto auto auto;
+  list-style: none;
+}
+
+.languages {
+  display: flex;
+  flex-direction: column;
+  list-style: none;
+  font-weight: 600;
+  margin-top: 30px;
+}
+
+.lang {
+  display: flex;
+  flex-direction: column;
+  padding-top: 50px;
+  padding-left: 50px;
+  margin-top: 20px;
+  min-width: 289px;
+  min-height: 289px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 800;
+  font-size: 20px;
+  line-height: 30px;
+  color: #fff;
+  background: #1c528a  url("images/figure1.png") 100% 100% no-repeat;
+}
+
+.lang1 {
+  display: flex;
+  flex-direction: column;
+  height: 10px;
+  left: 10px;
+  padding-top: 40px;
+  padding-bottom: 27px;
+  font-weight: 600;
+  font-size: 18px;
+  letter-spacing: -0.0525em;
+}
+
+.lang2 {
+  display: flex;
+  flex-direction: column;
+  height: 10px;
+  left: 65px;
+  top: 3891px;
+  padding-bottom: 27px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 18px;
+  line-height: 27px;
+  letter-spacing: -0.0525em;
+}
+
+.fram {
+  display: flex;
+  flex-direction: column;
+  padding-top: 50px;
+  padding-left: 50px;
+  min-width: 289px;
+  min-height: 289px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 800;
+  font-size: 20px;
+  line-height: 30px;
+  color: #fff;
+  background: #e2505c url("images/ic_framework.png") 100% 100% no-repeat;
+}
+
+.fram1 {
+  display: flex;
+  flex-direction: column;
+  width: 142px;
+  height: 10px;
+  padding-top: 40px;
+  padding-bottom: 20px;
+  font-size: 18px;
+  line-height: 27px;
+  letter-spacing: -0.0525em;
+}
+
+.fram2 {
+  display: flex;
+  flex-direction: column;
+  width: 142px;
+  height: 10px;
+  padding-bottom: 20px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 18px;
+  line-height: 27px;
+  letter-spacing: -0.0525em;
+}
+
+.skill1 {
+  display: flex;
+  flex-direction: column;
+  padding-top: 50px;
+  padding-left: 50px;
+  min-width: 289px;
+  min-height: 289px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 800;
+  font-size: 20px;
+  line-height: 30px;
+  color: #fff;
+  background: #febf52 url("images/ic_skills.png") 100% 100% no-repeat;
+}
+
+.skill2 {
+  display: flex;
+  flex-direction: column;
+  height: 10px;
+  margin-top: 20px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 18px;
+  line-height: 27px;
+  letter-spacing: -0.0525em;
+}
```

Comment:
- [ ] Some properties are used repeatedly in your code and this is really not consistent with CSS Best Practices because it makes the code a bit messy instead of keeping it DRY (Don't Repeat Yourself) :hammer_and_wrench: 
**_Properties like:_**
- display: flex;
- flex-direction: column/row;
- font-style
- font-family
- padding: 16px
- etc.

- [ ] I am suggesting creating a class for these items in the HTML document so that you only style them once. This will up your game as a pro, reduce your lines, and increase readability.

I really like your code, it's really clean :+1:

Reply:
Thank you for all the suggestions! I've worked through all the points you mentioned! I hope it's much better this time.